apps/storybook/src/Customization.mdx

```diff
@@ -135,13 +135,13 @@ as you see fit. For instance, if you'd like to change the color of the curve of
 | `--h5w-btnRaised--shadowColor`        | `gray`                                             | Box shadow color of raised buttons                        |
 | `--h5w-btnRaised-hover--shadowColor`  | `dimgray`                                          | Box shadow color of raised buttons on hover               |
 
-##### Selectors
+##### Selectors & menus
```

Comment:
`ExportMenu` now also uses the `--h5w-selector` variables. Didn't think it was worth renaming everything to `--h5w-menu`, especially since we use the term "menu" for the overflow menu even though it is technically more of a popover than a menu... Also, the `--h5w-selector` prefix is meant to describe the entire control, whereas ARIA considers that the "menu" is the container that pops open and contains the options. Anyways...

This is all a bit confusing, but the important thing is that `Selector` and `ExportMenu` share very similar styles (their "menus" both contain a number of clickable options), so they make use of the same variables (`--h5w-selector`) and classes (`.menu`, `.arrowIcon`, `.btnOption`, etc.) -- cf. `Toolbar.module.css`.

They no longer compose from the `popup` class (cf. `toolbar/utils.modules.css`), which is now used only by `DomainWidget`, `OverflowMenu` and `InteractionHelp` and has its own CSS variable (`--h5w-toolbar-popup--bgColor`, which defaults to `white` with 90% opacity, whereas `--h5w-selector-menu--bgColor` defaults to `white` with 100% opacity).